library/core/src/any.rs

```diff
@@ -1193,23 +1208,24 @@ impl<'a> Demand<'a> {
     }
 }
 
-#[unstable(feature = "provide_any", issue = "96024")]
-impl<'a> fmt::Debug for Demand<'a> {
+#[unstable(feature = "error_generic_member_access", issue = "99301")]
+impl<'a> fmt::Debug for Request<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Demand").finish_non_exhaustive()
+        f.debug_struct("Request").finish_non_exhaustive()
     }
 }
 
 ///////////////////////////////////////////////////////////////////////////////
 // Type tags
 ///////////////////////////////////////////////////////////////////////////////
 
-mod tags {
+#[unstable(feature = "error_generic_member_access", issue = "99301")]
```

Comment:
These feature tags should not be needed for things that are `pub(crate)`.

Reply:
d'oh!